pom.xml

```diff
@@ -4,12 +4,12 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.50</version>
+    <version>3.57</version>
     <relativePath />
   </parent>
   
   <properties>
-    <jenkins.version>2.150.3</jenkins.version>
+    <jenkins.version>2.164.3</jenkins.version>
```

Comment:
Just wondering the reason behind upping the Jenkins version? Looking at the stats there's still a non-insignificant number of people on 2.150 LTS branch (albeit not running latest version of the plugin) so I'd prefer to keep the support Jenkins version down as low as it can go until there's a need for a feature that comes with a newer version for instance.

Reply:
I thought 2.164 would be a good choice to go for, because it supports Java 11. I think we do not have a problem with java 11. And the amount of users, who are still using the latest plugin version with an older jenkins is pretty low (~100) compared to >10.000 according to: [jenkins version per plugin](https://stats.jenkins.io/plugin-installation-trend/jenkins-version-per-plugin-version.json)

Reply:
JDK11 support seems a good reason to me especially, as you say, given the version usage figures.